Procfile

```diff
@@ -0,0 +1 @@
+web: python3 -m server
```

Comment:
Can we start it using gunicorn rather than the test server

web: gunicorn --timeout=60 --workers=8 --threads=8 server:app